rl_chain/slates_chain.py

```diff
@@ -23,17 +25,17 @@
 from sentence_transformers import SentenceTransformer
 
 
-class Label:
+class SlatesLabel(base.Label):
```

Comment:
maybe class Label(base.Label)? (consistent with Event)

Reply:
yeah renamed and moved